### PR TITLE
[SQL] Add driver for drizzle

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: packages/plugin-client-drizzle/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const xata = getXataClient();
+const db = drizzle(xata);
+
+const drivers = pgTable('drivers', {
+  id: text('id').primaryKey(),
+  surname: text('surname'),
+  forename: text('forename')
+});
+
+const result = await db.select().from(drivers).execute();
```
Comment (tsg) [inline packages/plugin-client-drizzle/README.md:40]: nit: Could we show a where clause here?

File: packages/plugin-client-drizzle/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+## Model generation
+
+We offer an experimental model generation helper that will generate the models for you from your `tables` array in your `xata.ts` file.
```
Comment (tsg) [inline packages/plugin-client-drizzle/README.md:45]: Ooooh, cool

File: packages/plugin-client-drizzle/src/generate.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    case 'int':
+      return integer(column.name);
+    case 'float':
+      return decimal(column.name);
+    case 'datetime':
+      return date(column.name);
+    case 'multiple':
+    case 'object':
+    case 'vector':
+    case 'file[]':
```
Comment (tsg) [inline packages/plugin-client-drizzle/src/generate.ts:23]: Ah, now I understand you slack question.`vector` is "array of `real`" but not sure have that is represented. `file[]` is array of `jsonb`.

File: packages/plugin-client-drizzle/src/generate.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    case 'file':
+    case 'json':
```
Comment (tsg) [inline packages/plugin-client-drizzle/src/generate.ts:25]: file and json can be mapped to `jsonb`.

**Review by tsg — APPROVED** [review]
LGTM. I expect the model generation will break on objects and some of the more complex types, so I think in the docs we should recommend manually creating the schema for now.

File: packages/plugin-client-drizzle/README.md (hunk 1 above)
Comment (tsg) [reply packages/plugin-client-drizzle/README.md:40]: `  .where(eq('nationality', 'spanish'))` I think should work, with `eq` imported from `drizzle-orm`.

File: packages/plugin-client-drizzle/src/generate.ts (hunk 4 above)
Comment (SferaDev) [reply packages/plugin-client-drizzle/src/generate.ts:25]: I'll leave them as TODO, it's experimental and it's not clear how they'll map to the ORM

File: packages/plugin-client-drizzle/README.md (hunk 1 above)
Comment (SferaDev) [reply packages/plugin-client-drizzle/README.md]: Updated with a more complex example

**Pull request merged**